nilearn/decoding/decoder.py

```diff
@@ -1,43 +1,48 @@
 """High-level decoding object that exposes standard classification and
 regression strategies such as SVM, LogisticRegression and Ridge, with optional
-feature selection, integrated hyper-parameter selection
-and aggregation strategy in which the best models
-within a cross validation loop are averaged.
+feature selection, integrated hyper-parameter selection and aggregation strategy
+in which the best models within a cross validation loop are averaged.
+
+Also exposes a high-level method fREM which uses clustering and model
+ensembling to achieve state of the art performance
 """
 # Authors: Yannick Schwartz
 #          Andres Hoyos-Idrobo
 #          Binh Nguyen <tuan-binh.nguyen@inria.fr>
+#          Thomas Bazeille
 #
 # License: simplified BSD
 
 import itertools
 import warnings
 
 import numpy as np
+from joblib import Parallel, delayed
 from sklearn import clone
 from sklearn.base import RegressorMixin
 from sklearn.linear_model import LogisticRegression
 from sklearn.linear_model.base import LinearModel
 from sklearn.linear_model.ridge import RidgeCV, RidgeClassifierCV, _BaseRidgeCV
-from sklearn.model_selection import ParameterGrid, check_cv, LeaveOneGroupOut
+from sklearn.model_selection import ParameterGrid, check_cv, LeaveOneGroupOut, StratifiedShuffleSplit, ShuffleSplit
 from sklearn.preprocessing import LabelBinarizer
 from sklearn.svm import SVR, LinearSVC
 from sklearn.svm.bounds import l1_min_c
 from sklearn.utils.extmath import safe_sparse_dot
 from sklearn.utils.validation import check_is_fitted, check_X_y
-from joblib import Parallel, delayed
 
 try:
     from sklearn.metrics import check_scoring
 except ImportError:
     # for scikit-learn 0.18 and 0.19
     from sklearn.metrics.scorer import check_scoring
 
-from .._utils import CacheMixin
-from .._utils.cache_mixin import _check_memory
-from .._utils.param_validation import (_adjust_screening_percentile,
-                                       check_feature_screening)
-from ..input_data.masker_validation import check_embedded_nifti_masker
+from nilearn._utils import CacheMixin
+from nilearn._utils.cache_mixin import _check_memory
+from nilearn._utils.param_validation import (_adjust_screening_percentile,
+                                             check_feature_screening)
+from nilearn.input_data.masker_validation import check_embedded_nifti_masker
+from nilearn.regions.rena_clustering import ReNA
```

Comment:
These lines should be relatve import instead of absolute 
```suggestion
from .._utils import CacheMixin
from .._utils.cache_mixin import _check_memory
from .._utils.param_validation import (_adjust_screening_percentile,
                                                            check_feature_screening)
from ..input_data.masker_validation import check_embedded_nifti_masker
from ..regions.rena_clustering import ReNA
```

Reply:
The revised PEP8 guidelines stimulate absolute imports . Relative imports are warranted only if the absolute import is too verbose.

Reply:
Yes I thought nilearn was moving to absolute imports everywhere PR after PR ? Is that right ? should I change it for all the code I touch ?

Reply:
Don't change existing imports, it makes the changes noisy and code reviews tiresome.
If you add an import, or change one, for your PR, then switch to absolute imports.